examples/avr/assert/main.cpp

```diff
@@ -29,7 +29,7 @@ modm_abandon(const modm::AssertionInfo &info)
 	serialStream << IFS(" failed!\nAbandoning...\n");
 #endif
 
-	Leds::setOutput();
+	Leds::setOutput();asdfasdfasdf
```

Comment:
Now remove all the `asdfasdfasdf` and we are ready to merge!

Reply:
\*whip\*